src/mappings/generatedColumns.ts

```diff
@@ -41,7 +41,10 @@ export const extractEventArgs = (args: any[]) => {
  */
 export const extractValue = (obj: unknown, key: string): string => {
   if (obj) {
-    return obj[key] || obj[snakeToCamelCase(key)];
+    if (Object.keys(obj).includes(key)) {
```

Comment:
```suggestion
    if (key in obj) {
```
JS has the `in` operator. Would that work?

Reply:
I tried but it wasn't working so I switched to using keys